### ISSUE TITLE
Overview/Components: Update image tiles

### ISSUE THREAD

**Issue by thyhmdo** [carbon-design-system/carbon-website#3798]
### There are some new components that have not been updated on this page: https://carbondesignsystem.com/components/overview/
- Contained list
- Menu 
- Menu buttons
- Popover
- Progress bar
- Toggletip
- Treeview
-----
- [x] Create thumbnail images
- [x] Add tiles and thumbnail images 
- [x] Link them to according Component pages and update the system
- [x] Remove the Overflow menu thumbnail

**Comment by kingtraceyj** [comment]
Contained list
![list](https://github.com/carbon-design-system/carbon-website/assets/20582179/3e41ad6a-c103-4b4e-a718-33b589d2c3d9)

Menu
![menus](https://github.com/carbon-design-system/carbon-website/assets/20582179/e2c6ba18-af10-4d92-b4b4-f5ed043c1952)

Menu button
![buttons](https://github.com/carbon-design-system/carbon-website/assets/20582179/a99dee2f-7757-4f2f-a773-b3046571603f)

popover
![popover](https://github.com/carbon-design-system/carbon-website/assets/20582179/40b709dd-a979-4021-9525-a90ff748b324)

Progress
![progress-indicator](https://github.com/carbon-design-system/carbon-website/assets/20582179/da84622d-6cee-4f7e-a676-67b6d132c437)

Toggletip
(Still looking)

Treeview
![tree](https://github.com/carbon-design-system/carbon-website/assets/20582179/007254c3-acb4-4476-97c9-ddd9e72a180e)

**Comment by kingtraceyj** [comment]
Thumbnails from platform folder: https://ibm.box.com/s/mof5xw645yd74szh4j77e4jyss1bmesb

**Comment by kingtraceyj** [comment]
[Toggle tip image exists](https://next.carbondesignsystem.com/assets/components?q=toggletip) but i'm not sure where the original files is.  
![image](https://github.com/carbon-design-system/carbon-website/assets/20582179/74aa132d-6f02-4229-aba4-b3aee9bbb19f)

**Comment by laurenmrice** [comment]
- [ ] @jeanservaas Need to redo images with new specs from Marketing.

**Comment by thyhmdo** [comment]
Adding more:
- Ordered list
- Overflow menu
- Time picker

**Comment by jeanservaas** [comment]
Hi @alisonjoseph — i just realized I redid so many of these illustrations for platform, but they never made it up onto our site.

I just revamped all of our illustrations and added any missing/new ones. There may be some extras in here, for instance, I have list, but I also added unordered and ordered list images in case we want them. I also added a time picker even though that component doesn't have its own page.

[Carbon_component_overview_revised.zip](https://github.com/carbon-design-system/carbon-website/files/13879959/Carbon_component_overview_revised.zip)
